### DISCUSSION TITLE
Category and Category Heading tags should offer group_name, group_id variables

### DISCUSSION THREAD

**Discussion by nep** [ExpressionEngine/ExpressionEngine#2998] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
The Category and Category Heading tags do not return Group Name or Group ID.
https://docs.expressionengine.com/latest/channels/categories.html#variables
https://docs.expressionengine.com/latest/channels/category-heading.html#variables

**Describe the solution you'd like**
I'd like them to.
`{exp:channel:categories}
    {category_group_name} / {category_group_id}: {category_name}
{/exp:channel:categories}`

**Describe alternatives you've considered**
Seems like the alternative is to continue to suffer in silence.

**Teachability, Documentation, Adoption, Migration Strategy**
This will have no effect on current sites, require very little change in documentation, have virtually no risk of complications, and will be helpful to a very small number of grateful people  Edit: changed from group_name and group_id because of possible variable name collisions

- [x] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by rowansentesy** [comment]
+1 to this!

**Comment by robsonsobral** [comment]
Any chance of a PR, @nep?

**Comment by nep** [comment]
Not in the near future, sorry!

**Comment by intoeetive** [comment]
Added `{category_group_id}` and `{category_group_name}` variables; PR: https://github.com/ExpressionEngine/ExpressionEngine/pull/4667

User Guide: https://github.com/ExpressionEngine/ExpressionEngine-User-Guide/pull/918
